script/testing/oltpbench/test_case_oltp.py

```diff
@@ -123,8 +124,11 @@ def run_post_test(self):
                 report(self.publish_results, self.server_data,os.path.join(
                     os.getcwd(), "oltp_result",self.filename_suffix),self.publish_username,self.publish_password,self.query_mode)
         except:
-            traceback.print_exc(file=sys.stdout)
-            return ErrorCode.ERROR
+            if self.continue_on_error:
+                traceback.print_exc(file=sys.stdout)
+                return ErrorCode.ERROR
+            else:
+                raise
```

Comment:
I don't think this is actually where we want to make the changes. I think we want to make the change in `test_server.py` in the `run_test` function. It would also be nice if we could aggregate the errors so that we could print them all at the end.

Reply:
'It would also be nice if we could aggregate the errors so that we could print them all at the end. '

I think the aggregation of errors will lose some context information. There might some INFO or ERROR log that comes from the database or other processes before the error, which might be helpful for the developer to debug